helm/templates/01-operator-deployment.yaml

```diff
@@ -0,0 +1,259 @@
+        certified: "false"
+        containerImage: icr.io/cpopen/odlm@sha256:3076710c9891bfb573ea4744398d02977702e8cf24e9f5151f9607c4dc2f95db
+        createdAt: "2024-09-28T19:55:35Z"
+        description: The Operand Deployment Lifecycle Manager provides a Kubernetes
+          CRD-based API to manage the lifecycle of operands.
+        nss.operator.ibm.com/managed-operators: ibm-odlm
+        # olm.operatorGroup: common-service
+        # olm.operatorNamespace: cs-op
+        # olm.skipRange: '>=1.2.0 <4.3.8'
+        # olm.targetNamespaces: cs-op
+        operatorframework.io/properties: '{"properties":[{"type":"olm.gvk","value":{"group":"operator.ibm.com","kind":"OperandBindInfo","version":"v1alpha1"}},{"type":"olm.gvk","value":{"group":"operator.ibm.com","kind":"OperandConfig","version":"v1alpha1"}},{"type":"olm.gvk","value":{"group":"operator.ibm.com","kind":"OperandRegistry","version":"v1alpha1"}},{"type":"olm.gvk","value":{"group":"operator.ibm.com","kind":"OperandRequest","version":"v1alpha1"}},{"type":"olm.gvk","value":{"group":"operator.ibm.com","kind":"OperatorConfig","version":"v1alpha1"}},{"type":"olm.package","value":{"packageName":"ibm-odlm","version":"4.3.8"}}]}'
+        operators.openshift.io/infrastructure-features: '["disconnected"]'
+        operators.operatorframework.io/builder: operator-sdk-v1.32.0
+        operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+        productID: 068a62892a1e4db39641342e592daa25
+        productMetric: FREE
+        productName: IBM Cloud Platform Common Services
+        repository: https://github.com/IBM/operand-deployment-lifecycle-manager
+        support: IBM
+      creationTimestamp: null
+      labels:
+        app.kubernetes.io/managed-by: Helm
+        intent: projected
+        name: operand-deployment-lifecycle-manager
+        productName: IBM_Cloud_Platform_Common_Services
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: kubernetes.io/arch
+                operator: In
+                values:
+                - amd64
+                - ppc64le
+                - s390x
+      containers:
+      - args:
+        - -v=1
+        command:
+        - /manager
+        env:
+        - name: OPERATOR_NAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+        - name: WATCH_NAMESPACE
+          valueFrom:
+            configMapKeyRef:
+              key: namespaces
+              name: namespace-scope
+              optional: true
+        # this might be remove in non-olm env
+        - name: OPERATOR_CONDITION_NAME
+          value: operand-deployment-lifecycle-manager.v4.3.9
+        - name: NO_OLM
+          value: "true"
+        image: {{ .Values.operatorImage }}
```

Comment:
This should be `{{ .Values.imagePullPrefix}}/{{ .Values.imageRegistryNamespaceOperator}}/{{ .Values.operatorImage }}:latest`

Because we need to be able to configure the registry, and also CICD automation will update the tag/digest